README.md

```diff
@@ -111,7 +111,7 @@ $webMoney = new WebMoney\WebMoney(new CurlRequester);
 $request = new Request(Request::AUTH_LIGHT);
 $request->setRequestedWmid('REQUESTED WMID');
 
-$request->cert('FULL PATH TO THE CERTIFICATE FILE', 'FULL PATH TO THE CERTIFICATE KEY');
+$request->cert('FULL PATH TO THE CERTIFICATE FILE', 'FULL PATH TO THE CERTIFICATE KEY', '(OPTIONAL) PASSWORD');
```

Comment:
@baibaratsky  'FULL PATH TO THE CERTIFICATE FILE' and 'FULL PATH TO THE CERTIFICATE KEY' as parameters of cert()